test/preprocessing/readvtk/vtk2trixi.jl

```diff
@@ -0,0 +1,77 @@
+@testset verbose=true "vtk2trixi" begin
+    output_directory = "test/preprocessing/readvtk"
+
+    Random.seed!(1)
+    saved_ic = InitialCondition(; coordinates=rand(2, 12), velocity=rand(2, 12),
+                                density=1.0, mass=2.0,
+                                pressure=3.0)
```

Comment:
```suggestion
expected_ic = InitialCondition(; coordinates=rand(2, 12), velocity=rand(2, 12),
                               density=1.0, mass=2.0, pressure=3.0)
```
please change also the following `saved_*`